container/container.bzl

```diff
@@ -27,7 +27,7 @@ container = struct(
 )
 
 # The release of the github.com/google/containerregistry to consume.
-CONTAINERREGISTRY_RELEASE = "v0.0.26"
```

Comment:
Other part of this PR LGTM. Only this one thing: 
Any reason we want to change the format of version number, i.e. "v<version>" to "<version>"? 

As you can see we have the single letter 'v' repeated multiple times in the URLs below, which doesn't look more readable for me.

Reply:
I moved this into the urls in order to make the `strip_prefix = "containerregistry-" + CONTAINERREGISTRY_RELEASE` line cleaner.

I've changed it back to the old way. Please let me know if you'd like me to make any other changes.